src/common/util.js

```diff
@@ -1,3 +1,5 @@
+import { PluralForm } from "./pluralform.js";
```

Comment:
Minor nit, can you use `export default` in pluralform.js and `import PluralForm from "./pluralform.js";` here? Since there is only one export from that file, we can use default exports.

Reply:
Ok, done.